Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "miette"
-version = "4.6.1-alpha.0"
+version = "4.4.8"
```

Comment:
I think this should not be in the commit

Reply:
Whoopsi.